discovery-provider/src/utils/seed.py

```diff
@@ -0,0 +1,94 @@
+import argparse
```

Comment:
I found this useful to seed the database with a large sample of seed data to test queries against. I could see it being useful for testing the API in the future and doing query performance metrics and tests, but ultimately a standalone file meant to be run alone

Reply:
This is sweet! Would be nice to link whatever resources you were looking at for pandas etc for future expansion here. Maybe this file wants to live in /scripts rather than src?